lib/src/store.dart

```diff
@@ -159,8 +159,8 @@ class Store<State> {
   Reducer<State> reducer;
 
   final StreamController<State> _changeController;
-  State _state;
-  List<NextDispatcher> _dispatchers;
+  late State _state;
```

Comment:
If you move the `_state = initialState` into the initializer list, this field does not need to be late.